src/somesy/cli/sync.py

```diff
@@ -94,6 +94,19 @@ def sync(
         help="Custom pom.xml (Java Maven) file path (default: pom.xml)",
         **existing_file_arg_config,
     ),
+    no_sync_mkdocs: bool = typer.Option(
+        None,
+        "--no-sync-mkdocs",
+        "-X",
+        help="Do not sync mkdocs.yml file (default: False)",
+    ),
+    mkdocs_file: Path = typer.Option(
+        None,
+        "--mkdocs-file",
+        "-x",
```

Comment:
`-X/-x` is for POM xml, let's make this  `-D/-d` for 'docs'? still free.

Also let's reserve `-R/-r` for Rust Cargo (C is used already for codemeta)

Reply:
Unless we want to add `R` the language, then keeping `R` for it would make also sense...

annoying that there's just 26 letters :D

Reply:
copy paste mistake :')